packages/web-components/fast-components/docs/api-report.md

```diff
@@ -18,14 +18,18 @@ import { Dialog } from '@microsoft/fast-foundation';
 import { Direction } from '@microsoft/fast-web-utilities';
 import { Divider } from '@microsoft/fast-foundation';
 import { Flipper } from '@microsoft/fast-foundation';
+import { Listbox } from '@microsoft/fast-foundation';
```

Comment:
I think we should have a seperate pr/spec for listbox if we're calling it the "FAST listbox" which implies it is a generic building block component.